document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 

```

Comment:
Testing a reply.

---
In reply to: [59280130](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r59280130) [](ancestors = 59280130)